src/NormalizingFlows.jl

```diff
@@ -1,5 +1,48 @@
 module NormalizingFlows
 
-# Write your package code here.
+using Bijectors
+using Optimisers
+using ProgressMeter
+using LinearAlgebra
+using Zygote
+using DiffResults
+import AbstractDifferentiation as AD
+
+include("train.jl")
+include("elbo.jl")
+
+function NF(
+    vo,                                      # elbo, likelihood, f-div, STL, etc.. (how do we deal with this? it would require different input)
+    flow::Bijectors.TransformedDistribution, # flow = T q₀, where T <: Bijectors.Bijector, q₀ reference dist that one can easily sample and compute logpdf
```

Comment:
> We spoke about how it might be a good idea to just remove this type-annotation since we don't use any functionality in this call that is specific to flow being of type Bijectors.TransformedDistribution, right?

Yes. I left this type-annotation only because the current `vo` implemented requires `flow::Bijectors.TransformedDistribution`. Will remove it!